### DISCUSSION TITLE
Copilot maybe shouldn't be able to do this...

### DISCUSSION THREAD

**Discussion by Neppkun** [community/community#7449] [Copilot Conversations]
Was seeing how Copilot would complete some sentences and apparently you can make it racist with a little effort.

![image](https://user-images.githubusercontent.com/57060018/140700415-e2e17891-015c-4b11-be92-db65773a2557.png)

**Comment by orph** [comment]
Can you mail this report to copilot-safety@github.com? Thank you.
